lamp/channels.php

```diff
@@ -38,6 +39,10 @@
                     }
                     ?>
                 </h2>
+                <?php
+                    if ($is_instructor)
+                        ?><h2>INSTRUCTOR VIEW</h2><?php;
```

Comment:
We should make this message more user-friendly. Perhaps just a badge/pill in the sidebar or navbar will work.

Reply:
Feel like we can just change this into still instructor view but with lowercase and different styling.